yellowbrick/features/pcoords.py

```diff
@@ -231,3 +294,82 @@ def finalize(self, **kwargs):
         # Set the legend and the grid
         self.ax.legend(loc='best')
         self.ax.grid()
+
+
+##########################################################################
+## Normalized Static Parallel Coordinates Visualizer
+##########################################################################
+
+class NormalizedParallelCoordinates(ParallelCoordinates):
```

Comment:
I think you're right, this should just be an argument to the ParallelCoordinates visualizer, the subclass doesn't add much exception confusion!